odl/tomo/geometry/geometry.py

```diff
@@ -154,6 +155,18 @@ def ndim(self):
         return self._ndim
 
     @property
+    def implementations(self):
```

Comment:
Alright, we should actually not use private attributes too much externally. Then s.th. like `impl_cache` would sound even more mysterious.